src/client/app/view-history-report/view-history-report.html

```diff
@@ -9,40 +9,40 @@
   <div class="row">
     <div class="col-md-3">
 
-          <div class="input-group">
-            <input type="text" class="form-control" ng-model="vm.tripLeader" width="30" />
-          </div>
+      <div class="input-group">
+        <input type="text" class="form-control" ng-model="vm.tripLeader" width="30" />
+      </div>
 
     </div>
     <div class="col-md-3">
       <button ng-click="vm.getHistoryReportByTripLeader()" class="btn btn-default" type="submit">Search</button>
     </div>
   </div>
 
-  
+
   <div>
     <div class="row">
       <div class="col-md-12">
         <table id="gear_request_table" class="table table-bordered">
           <thead>
-            <tr>
-              <th>Item Name</th>
-              <th>Start Date</th>
-              <th>End Date</th>
-              <th>Total Quantity</th>
-              <th>Status</th>
-              <th>Care Maintenance</th>
-            </tr>
+          <tr>
```

Comment:
Minor thing: the convention I am familiar with is to indent the contents within the <thead> and <tbody> tags, as it was in the previous version of this file. I am seeing this convention upheld in other html files too. Examples: https://github.com/SEGR-SEATTLEU/FALL-2016/blob/development/src/client/app/view-history-report/view-history-report.html, https://github.com/SEGR-SEATTLEU/FALL-2016/blob/development/src/client/app/gear-availability-report/gear-availability-report.html.